contrib/parseq-tracevis-server/pom.xml

```diff
@@ -101,7 +101,6 @@
 					<descriptors>
 						<descriptor>assembly-descriptor.xml</descriptor>
 					</descriptors>
-					<finalName>parseq-tracevis-server</finalName>
```

Comment:
Interesting, so where is this "*-with-dependencies.jar" name defined?

Reply:
There is another file named "assembly-descriptor.xml" which defines the dependencies.jar.